src/euler.jl

```diff
@@ -13,7 +13,9 @@ function euler!(Y, u, W::SamplePath, P)
 
     for i in 1:N-1
         yy[.., i] = y
-        y = y + b(tt[i], y, P)*(tt[i+1]-tt[i]) + σ(tt[i], y, P)*(ww[.., i+1]-ww[..,i])
+        factor = σ(tt[i], y, P)
+        isa(factor,UniformScaling) && ( factor = factor.λ )
+        y = y + b(tt[i], y, P)*(tt[i+1]-tt[i]) + factor*(ww[.., i+1]-ww[..,i])
     end
```

Comment:
The problem was not multiplying by a UniformScaling, it was adding to the other terms.